src/components/layout/navbar.js

```diff
@@ -1,34 +1,114 @@
 import Link from 'next/link';
 import { Container } from '../ui/containers';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { PrimaryButton } from '../ui/buttons';
-import { H2, NavbarLink } from '../ui/typography';
+import { H3, NavbarLink, LgText } from '../ui/typography';
+import { CloseIcon, HamburgerIcon } from '../ui/icons';
 
 const Navbar = () => {
   const [isOpen, setIsOpen] = useState(false);
+  // toggle the submenu in mobile on click
+  const [isSubMenuOpen, setIsSubMenuOpen] = useState([]);
 
   const navigation = [
-    { name: 'Home', href: '/' },
-    { name: 'About', href: '/about' },
-    { name: 'Services', href: '/services' },
+    {
+      name: 'Home',
+      href: '/',
+    },
+    {
+      name: 'About',
+      href: '/about',
+      subItem: [
+        {
+          name: 'About our team',
+          href: '/about/team',
+        },
+        {
+          name: 'History Timeline',
+          href: '/about/history',
+        },
+      ],
+    },
+    {
+      name: 'Services',
+      href: '/services',
+      subItem: [
+        {
+          name: 'Software Engineering',
+          href: '/services/software-engineering',
+        },
+        {
+          name: 'Marketing',
+          href: '/services/Marketing',
+        },
+      ],
+    },
   ];
+  // useEffect(() => {
+  //   console.log(isSubMenuOpen);
+  // }, [isSubMenuOpen]);
```

Comment:
Commented code here that we might want to remove

Reply:
Removed.